Modix.Data/Models/Core/MessageEntity.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
+using Modix.Data.Utilities;
+
+namespace Modix.Data.Models.Core
+{
+    public class MessageEntity
```

Comment:
I think this should at least be `GuildMessageEntity`, in the same fashion as we track `GuildRoleEntity` and `GuildChannelEntity`. Also, I think maybe `GuildMessageLogEntity` or `GuildMessageInfoEntity` or something similar.